Spanner/tests/Unit/SpannerClientTest.php

```diff
@@ -60,6 +60,7 @@ class SpannerClientTest extends TestCase
 
     private $client;
     private $connection;
+    private $instance;
```

Comment:
I thought the `$instance` could be useful to the future tests. Since it is not used any function for now, will move it inside the `testSpannerClientDatabaseRole` function for now.